lib/client/twofa/twofa.go

```diff
@@ -143,7 +143,8 @@ func getCertsFromServer(
 	if err != nil {
 		return nil, nil, nil, err
 	}
-	loginResp.Body.Close() //so that we can reuse the channel
+	loginResp.Body.Close()                  //so that we can reuse the channel
+	io.Copy(ioutil.Discard, loginResp.Body) // We also need to read ALL of the body
```

Comment:
Discarding the data should be done before closing, right?

Reply:
woops yes.